lab_1_keywords_tfidf/main.py

```diff
@@ -17,7 +19,11 @@ def clean_and_tokenize(text: str) -> Optional[list[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str):
+        return None
```

Comment:
you did change the order of branches here as I asked, which is great, but it would also be great if you did so in all the other functions as well